.github/workflows/release.yml

```diff
@@ -106,10 +106,6 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       fail-fast: false
-      matrix:
-        include:
-          - test_type: pulp3-source-centos7-fips
-          - test_type: pulp3-source-centos8-fips
```

Comment:
you need to remove the entire job, 
`steps` below without `test_type` will fail/test nothing